src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/HandlerFactory.java

```diff
@@ -74,6 +74,8 @@ public HandlerFactory() {
         register(TokenTypes.INDEX_OP, IndexHandler.class);
         register(TokenTypes.LITERAL_SYNCHRONIZED, SynchronizedHandler.class);
         register(TokenTypes.LAMBDA, LambdaHandler.class);
+        register(TokenTypes.ANNOTATION_DEF, ClassDefHandler.class);
+        register(TokenTypes.ANNOTATION_FIELD_DEF, AnnotationDefHandler.class);
```

Comment:
>AnnotationDefHandler looks like it is coded like MemberDefHandler.

Yes I based the field checking on member def handler.

The only case that the current `ClassDefHandler` is not solving for Annotation definition is line wrapping as you pointed out it in https://github.com/checkstyle/checkstyle/pull/4217#discussion_r111635354